symfony/client/src/orangehrmPimPlugin/pages/employee/EmployeeImmigration.vue

```diff
@@ -0,0 +1,244 @@
+      http,
+      showPaginator,
+      currentPage,
+      isLoading,
+      total,
+      pages,
+      pageSize,
+      execQuery,
+      items: response,
+    };
+  },
+
+  data() {
+    return {
+      headers: [
+        {name: 'type', slot: 'title', title: 'Document', style: {flex: 1}},
+        {name: 'number', title: 'Number', style: {flex: 1}},
+        {name: 'countryCode', title: 'Issued By', style: {flex: 1}},
+        {name: 'issuedDate', title: 'Issued Date', style: {flex: 1}},
+        {name: 'expiryDate', title: 'Expiry Date', style: {flex: 1}},
+        {
+          name: 'actions',
+          slot: 'action',
+          title: 'Actions',
+          style: {flex: 1},
+          cellType: 'oxd-table-cell-actions',
+          cellConfig: {
+            delete: {
+              onClick: this.onClickDelete,
+              component: 'oxd-icon-button',
+              props: {
+                name: 'trash',
+              },
+            },
+            edit: {
+              onClick: this.onClickEdit,
+              props: {
+                name: 'pencil-fill',
+              },
+            },
+          },
+        },
+      ],
+      checkedItems: [],
+      showSaveModal: false,
+      showEditModal: false,
+      editModalState: null,
+    };
+  },
+
+  methods: {
+    onClickDeleteSelected() {
+      const ids = this.checkedItems.map(index => {
+        return this.items?.data[index].id;
+      });
+      this.$refs.deleteDialog.showDialog().then(confirmation => {
+        if (confirmation === 'ok') {
+          this.deleteItems(ids);
+        }
+      });
+    },
+    onClickDelete(item) {
+      this.$refs.deleteDialog.showDialog().then(confirmation => {
+        if (confirmation === 'ok') {
+          this.deleteItems([item.id]);
+        }
+      });
+    },
+    deleteItems(items) {
+      if (items instanceof Array) {
+        this.isLoading = true;
+        this.http
+          .deleteAll({
+            ids: items,
+          })
+          .then(() => {
+            return this.$toast.deleteSuccess();
+          })
+          .then(() => {
+            this.isLoading = false;
+            this.resetDataTable();
+          });
+      }
+    },
+    async resetDataTable() {
+      this.checkedItems = [];
+      await this.execQuery();
+    },
+    onClickAdd() {
+      this.showEditModal = false;
+      this.editModalState = null;
+      this.showSaveModal = true;
+    },
+    onClickEdit(item) {
+      this.showSaveModal = false;
+      this.editModalState = item;
+      this.showEditModal = true;
+    },
+    onSaveModalClose() {
+      this.showSaveModal = false;
+      this.resetDataTable();
+    },
+    onEditModalClose() {
+      this.showEditModal = false;
+      this.editModalState = null;
+      this.resetDataTable();
+    },
+  },
+
+  computed: {
+    isDisabled() {
+      return this.showSaveModal || this.showEditModal;
+    },
+  },
+};
+</script>
+
+<style src="./employee.scss" lang="scss" scoped></style>
```

Comment:
@RajithaKumara no issue if used with class name and separate styles or to use existing styles.

Reply:
Let me explain the exact case. may be wrong. will see,
`.orangehrm-employee-container` this class seems only used in Add Employee screen. but since the class in `./employee.scss` and it include as `<style src="./employee.scss" lang="scss" scoped></style>`  (with scoped) in multiple components `.orangehrm-employee-container` class added as mutiple times to app.css.

[app.md](https://github.com/orangehrm/orangehrm/files/6842358/app.md)

```css
.orangehrm-employee-container[data-v-405284e4] {
  display: flex;
}
@media (min-width: 0px) {
  .orangehrm-employee-container[data-v-405284e4] {
    flex-direction: column;
  }
}
@media (min-width: 800px) {
  .orangehrm-employee-container[data-v-405284e4] {
    flex-direction: row;
  }
}
.orangehrm-employee-container .orangehrm-employee-image[data-v-405284e4] {
  flex: 1 1 25%;
  padding: 1rem;
  text-align: center;
}
.orangehrm-employee-container .orangehrm-employee-form[data-v-405284e4] {
  flex: 1 1 75%;
}

//////////////////////////////////////////////////

.orangehrm-employee-container[data-v-01e3a037] {
  display: flex;
}
@media (min-width: 0px) {
  .orangehrm-employee-container[data-v-01e3a037] {
    flex-direction: column;
  }
}
@media (min-width: 800px) {
  .orangehrm-employee-container[data-v-01e3a037] {
    flex-direction: row;
  }
}
.orangehrm-employee-container .orangehrm-employee-image[data-v-01e3a037] {
  flex: 1 1 25%;
  padding: 1rem;
  text-align: center;
}
.orangehrm-employee-container .orangehrm-employee-form[data-v-01e3a037] {
  flex: 1 1 75%;
}

//////////////////////////////////////////////////

.orangehrm-employee-container[data-v-60632ff5] {
  display: flex;
}
@media (min-width: 0px) {
  .orangehrm-employee-container[data-v-60632ff5] {
    flex-direction: column;
  }
}
@media (min-width: 800px) {
  .orangehrm-employee-container[data-v-60632ff5] {
    flex-direction: row;
  }
}
.orangehrm-employee-container .orangehrm-employee-image[data-v-60632ff5] {
  flex: 1 1 25%;
  padding: 1rem;
  text-align: center;
}
.orangehrm-employee-container .orangehrm-employee-form[data-v-60632ff5] {
  flex: 1 1 75%;
}

//////////////////////////////////////////////////

.orangehrm-employee-container[data-v-7b30d5e0] {
  display: flex;
}
@media (min-width: 0px) {
  .orangehrm-employee-container[data-v-7b30d5e0] {
    flex-direction: column;
  }
}
@media (min-width: 800px) {
  .orangehrm-employee-container[data-v-7b30d5e0] {
    flex-direction: row;
  }
}
.orangehrm-employee-container .orangehrm-employee-image[data-v-7b30d5e0] {
  flex: 1 1 25%;
  padding: 1rem;
  text-align: center;
}
.orangehrm-employee-container .orangehrm-employee-form[data-v-7b30d5e0] {
  flex: 1 1 75%;
}
```

Reply:
Have to check this. might be resolved when building for production, but not sure.